build_stats.sh

```diff
@@ -0,0 +1,23 @@
+#! /bin/sh
+
+INST_DIR=$HOME/R-inst
+mkdir -p ${INST_DIR}
+
+# Download a recent version of R (must be 3.3.1 or later).
+wget https://cran.r-project.org/src/base/R-3/R-3.3.2.tar.gz
+tar -xvzf R-3.3.2.tar.gz
+cd R-3.3.2
+
+# Build R.
+./configure --prefix=${INST_DIR} --enable-R-shlib
+make
+make install
+
+cd ..
+
```

Comment:
My normal would be `echo "export PATH=$PATH:..." >> ~/.bashrc` otherwise the new `PATH` will be lost when the shell is closed, that won't work for everyone, I think?